packages/twenty-front/project.json

```diff
@@ -67,7 +67,7 @@
     "test": {},
     "storybook:build": {
       "options": {
-        "env": { "NODE_OPTIONS": "--max_old_space_size=6000" }
+        "env": { "NODE_OPTIONS": "--max_old_space_size=8000" }
```

Comment:
This is a lot of RAM, I have already investigated it and the solution is to split the project into pieces OR lazy loading strategy on some app branches (already done for Settings)